src/engine/basics/PointFunctions.cxx

```diff
@@ -241,7 +242,24 @@ NeighbourNodesPosition PointFunctions::getNeighborPositionToOrigin(const Point &
     }
     break;
   }
-
-  LOG(LOG_ERROR) << "No position found! This should never happen";
-  return NeighbourNodesPosition::CENTER;
+  //assert(false); // this should never happen!
 }
+
+bool PointFunctions::isNeighborOf(Point coordinateOne, Point coordinateTwo)
```

Comment:
why not just implement this in Point class ? not sure if it's a class or struct

Reply:
I would also prefer this @Bogi79

Reply:
I agree on that one.

Consider moving complete PointFunctions to the Point file.
The proposed one can be a member of the Point class. The rest may be out-of-class functions or if you prefer static functions of the Point class.